corehq/apps/toggle_ui/views.py

```diff
@@ -87,6 +96,14 @@ def page_context(self):
             )
         return context
 
+    def call_save_fn(self, changed_entries, new_entries):
+        if self.static_toggle.save_fn is None:
+            return
+        for entry in changed_entries:
+            if entry.startswith(NAMESPACE_DOMAIN):
+                domain = entry.split(":")[-1]
+                self.static_toggle.save_fn(domain, entry in new_entries)
```

Comment:
ah, I see.  The second parameter is a boolean which is meant to reflect whether the toggle has been turned on or off - the entry is just a string like "domain:opm"